lib/Runtime/Language/DynamicProfileInfo.h

```diff
@@ -36,7 +37,7 @@ namespace IR
     BailOutKind EquivalentToMonoTypeCheckBailOutKind(BailOutKind kind);
 }
 
-#if ENABLE_DEBUG_CONFIG_OPTIONS
+#if ENABLE_DEBUG_CONFIG_OPTIONS || defined(REJIT_STATS)
```

Comment:
Part of the goal for this change is so that for e.g. crawling, this code can be enabled in non-debug builds.